tests/contract/test_resource_client.py

```diff
@@ -85,15 +85,21 @@
         "b": {"type": "number"},
         "c": {"type": "number"},
         "d": {"type": "number"},
+        "e": {"type": "string"},
     },
     "readOnlyProperties": ["/properties/b"],
     "createOnlyProperties": ["/properties/c"],
-    "primaryIdentifier": ["/properties/c"],
+    "primaryIdentifier": ["/properties/b"],
     "writeOnlyProperties": ["/properties/d"],
-    "propertyTransform": {"/properties/a": '$join([a, "Test"])'},
+    "propertyTransform": {
+        "/properties/a": '$join([a, "Test"])',
+        "/properties/c": "$power(2, 2)",
+        "/properties/e": '$join([e, "Test"]) $OR $join([e, "Value"])',
```

Comment:
is something like
```
"/properties/e": '$join([e, "T$OR"]) $OR $join([e, "Value"])',
```

currently supported? if not we should probably document the limitations of using the OR support

Reply:
```$join([e, "T$OR"])``` is not supported. I am not sure I understand the use case when this is valid. Could you elaborate on that. I can add this to drift documentation.

Reply:
i think just letting people know that $OR is reserved and can't be used in transformations other than to specify multiple would be good